lib/waitforapp.js

```diff
@@ -8,6 +8,12 @@ var SEARCH_TIMEOUT = 10000;
  */
 var WAIT_BETWEEN_CHECKS = 250;
 
+function assertDriverIsSync(client) {
+  if (!client.isSync) {
+    throw Error('No longer support async driver. Please use sync driver.');
```

Comment:
I think we want the `Error` constructor here (ie `new Error`) even though we didn't have the `new` keyword before